### DISCUSSION TITLE
Automated machine translation of entire locale sites? (decoupling machine translation processes from the admin interface)

### DISCUSSION THREAD

**Discussion by leonh** [wagtail/wagtail-localize#372] [Ideas]
Is there a recipe that I can follow to machine translate an entire locale site maybe via a management command or python script? 

Currently to my (limited) understanding machine translation works on a page-by-page basis and is designed to be driven by user interaction in the admin edit pages. It seems that there is a possibility to modify edit_translation.py and decouple from react and the admin ui to allow this?

**Comment by kaedroho** [comment]
Thanks @leonh 

You're right, it only supports one page at a time at the moment. I think creating a management command for this would be a good idea, and we might have bulk actions in Wagtail soon so it might be possible to implement it as one of those too.
